pkgs/tools/networking/grpcui/default.nix

```diff
@@ -0,0 +1,23 @@
+{ buildGoModule, fetchFromGitHub, lib }:
+
+buildGoModule rec {
+  version = "0.2.0";
+  name = "grpcui-${version}";
```

Comment:
Please use `pname = "grpcui";` instead of `name`. `name` is automatically implied if you set `pname` and `version`.

Reply:
Wow nice! I had no idea about this.